cpp/main.cc

```diff
@@ -514,14 +553,19 @@ int main(int argc, char **argv) {
           TString trees = "/ceph/cms"+baseDir+"/"+year+"/"+sample_name+"_"+sample_prod[sample][year][d]+"_"+dataformat+"_"+version+"/"+"tree_*.root"; // Local access
           //TString trees = "davs://redirector.t2.ucsd.edu:1095"+baseDir+"/"+year+"/"+sample_name+"_"+sample_prod[sample][year][d]+"_"+dataformat+"_"+version+"/"+"tree_*.root"; // Global access
 
+          // inverted 
+          if (enrich_DY){
+            trees = "/ceph/cms"+invDir+"/"+year+"/"+sample_name+"_"+sample_prod[sample][year][d]+"/"+"tree_*9.root";
```

Comment:
Why the `9` before the `.root` here. Does this imply that not all of the files have been used for the production of the samples?

Reply:
Fixed